libcontainer/cgroups/utils.go

```diff
@@ -21,49 +21,31 @@ const cgroupNamePrefix = "name="
 
 // https://www.kernel.org/doc/Documentation/cgroups/cgroups.txt
 func FindCgroupMountpoint(subsystem string) (string, error) {
-	f, err := os.Open("/proc/self/mountinfo")
+	mounts, err := mount.GetMounts()
```

Comment:
I think the reason @LK4D4 was parsing /proc/self/mountinfo directly as it was more efficient than this call. 
ping @LK4D4